packages/client/src/pages/manage-users.vue

```diff
@@ -180,6 +186,7 @@
           <template #body-cell-pay-off="{ row }">
             <q-td>
               <chip-button
+                :disable="willBeDeleted(row)"
```

Comment:
Is your usage of both `disabled` and `disable` on the same component on purpose?

Reply:
`disabled` came yesterday from #54. `disable` is what I added originally. I haven't rebased the PR, so I didn't get to manage this difference. But, `disabled` is the less correct one as it doesn't disable the hover effects, click effects, etc. So, I'll fix the previous faulty usage.